src/address/constants.ts

```diff
@@ -1,3 +1,8 @@
+export enum SignatureType {
+  SECP256K1 = 1,
+  BLS = 3,
```

Comment:
Enum skipped type for 2

Reply:
There is no type 2 for signatures. That is why there is no type there.